LLVM 21 complains here about something in the UMA_HISTOGRAM_ENUMERATION
not being a constant expression, this code effectively doesn't do
anything and is only used for reporting so disable the code

diff --git a/src/3rdparty/chromium/sandbox/policy/linux/sandbox_linux.cc b/src/3rdparty/chromium/sandbox/policy/linux/sandbox_linux.cc
index f793d20b40..c46ed56ea9 100644
--- a/src/3rdparty/chromium/sandbox/policy/linux/sandbox_linux.cc
+++ b/src/3rdparty/chromium/sandbox/policy/linux/sandbox_linux.cc
@@ -671,6 +671,7 @@ bool SandboxLinux::EngageNamespaceSandboxInternal(bool from_zygote) {
 }
 
 void SandboxLinux::ReportLandlockStatus() {
+#if 0
   LandlockState landlock_state = LandlockState::kUnknown;
   const int landlock_version =
       landlock_create_ruleset(nullptr, 0, LANDLOCK_CREATE_RULESET_VERSION);
@@ -698,6 +699,7 @@ void SandboxLinux::ReportLandlockStatus() {
   }
 
   UMA_HISTOGRAM_ENUMERATION("Security.Sandbox.LandlockState", landlock_state);
+#endif
 }
 
 }  // namespace policy
diff --git a/src/3rdparty/chromium/content/browser/webid/federated_auth_request_impl.cc b/src/3rdparty/chromium/content/browser/webid/federated_auth_request_impl.cc
index 9ce445f050..0836f65ffa 100644
--- a/src/3rdparty/chromium/content/browser/webid/federated_auth_request_impl.cc
+++ b/src/3rdparty/chromium/content/browser/webid/federated_auth_request_impl.cc
@@ -2501,7 +2501,7 @@ void FederatedAuthRequestImpl::ShowModalDialog(DialogType dialog_type,
   // dialog?
   dialog_type_ = dialog_type;
   config_url_ = idp_config_url;
-  UMA_HISTOGRAM_ENUMERATION("Blink.FedCm.Popup.DialogType", dialog_type_);
+  //UMA_HISTOGRAM_ENUMERATION("Blink.FedCm.Popup.DialogType", dialog_type_);
 
   WebContents* web_contents = request_dialog_controller_->ShowModalDialog(
       url_to_show, rp_mode_,
